### PR TITLE
[EASY] solve 595_big_countries

### PR DESCRIPTION
**Pull request by DONGHO-KIM-DBA**
[EASY] solve 595_big_countries

### CODE REVIEW

**Comment by DONGHO-KIM-DBA** [comment]
[EASY] solved 1179_reformat_department_table

File: leetcode/easy/595_big_countries/dongho.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--https://leetcode.com/problems/big-countries/submissions/
+--Runtime: 937 ms, faster than 77.97% of MS SQL Server online submissions for Big Countries.
+--Memory Usage: 0B, less than 100.00% of MS SQL Server online submissions for Big Countries.
+
+SELECT name, population, area
+  FROM World
+ WHERE (population > 25000000 or area > 3000000)
```
Comment (saekomkr) [inline leetcode/easy/595_big_countries/dongho.sql]: 깔끔하네요 ㅋㅋㅋ :+1:

Comment (DONGHO-KIM-DBA) [reply leetcode/easy/595_big_countries/dongho.sql]: SQL 입문용 문제도 섞여있네요.~

File: leetcode/easy/1179_reformat_department_table/dongho.sql [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	   [Sep] AS Sep_Revenue, 
+	   [Oct] AS Oct_Revenue, 
+	   [Nov] AS Nov_Revenue, 
+	   [Dec] AS Dec_Revenue
+  FROM Department
+ PIVOT 
+ (
+	MAX(Revenue)
+	FOR month IN (Jan, Feb, Mar, Apr, May, Jun, Jul, Aug, Sep, Oct, Nov, Dec) 
+ ) AS MonthsRevenue
```
Comment (netty1010) [inline leetcode/easy/1179_reformat_department_table/dongho.sql]: 👍

**Pull request merged**